src/isomorphic/classic/element/ReactCurrentOwner.js

```diff
@@ -11,20 +11,6 @@
 
 'use strict';
 
-/**
- * Keeps track of the current owner.
- *
- * The current owner is the component who should own any components that are
- * currently being constructed.
- */
-var ReactCurrentOwner = {
-
-  /**
-   * @internal
-   * @type {ReactComponent}
-   */
-  current: null,
-
-};
+var ReactCurrentOwner = require('react-current-owner');
```

Comment:
wasn’t there some work to reign in the build system somewhere?

Updated with that and now `grunt build` works correctly, the browser examples each work, and I can `npm link` this into my custom renderer and require `react` directly instead of `react/lib/ReactIsomorphic`.